src/helpers/sdks.js

```diff
@@ -97,4 +97,34 @@ module.exports = {
     }
     return Promise.resolve(['iOS SDK', 'N/A']);
   },
+
+  getWindowsSDKInfo: () => {
+    let info;
+    if (utils.isWindows) {
+      return utils
+        .run('reg query HKLM\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\AppModelUnlock')
+        .then(out => {
+          const devModeLines = out
+            .split(/[\r\n]/g)
+            .slice(1)
+            .filter(x => x !== '');
+          const devModeMap = devModeLines.reduce((m, o) => {
+            const values = o.match(/[^\s]+/g);
+            m[values[0]] = values[2];
+            return m;
```

Comment:
I'm assuming this is where values like `0x1` are coming from - I also assume these are hex representation of a binary value, I think it makes more sense to coerce this to boolean?

Reply:
they will usually be boolean although they could be other types (just numbers, or strings, etc.). I'll special case 0x0 and 0x1 since those are how these regkeys are most often going to be used

Reply:
BTW @tabrindle when I changed this to return boolean true/false, the properties get removed during formatting by the clean function:

```js
function clean(data, options) {
  utils.log('trace', 'clean', data);
  return Object.keys(data).reduce((acc, prop) => {
    if (
      (!options.showNotFound && data[prop] === 'Not Found') ||
      data[prop] === 'N/A' ||
      data[prop] === undefined ||
      Object.keys(data[prop]).length === 0
    )
```
Because when it gets to process the true/false values, Object.keys(true).length === 0 so it doesn't get added to the map. 
To work around this I'm using a String with the boolean value instead of a real boolean.

Reply:
ah yes, apologies, I didn't remember this. I think this is the first time we've used a boolean. Would enabled/disabled be semantically correct for this usage? The idea of making string booleans makes me itchy :-|

Reply:
yeah that works :)